perceval/rendering/circuit/renderer.py

```diff
@@ -43,12 +43,6 @@ class PortPos:
     def __init__(self, x, y, fixed=True):
```

Comment:
I think you can remove the parameter `fixed=True`.

Also, `class PortPos` was meant to follow a herald "Port" moving along it's mode, following permutations. It seems `collect_herald_info` does it for both input and output, and replaces the need for any kind of "changing x and y on the fly". Is that correct?

Reply:
Sure, it's not needed, I can remove it.